x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/ComputeListener.java

```diff
@@ -188,6 +189,27 @@ ActionListener<Void> acquireAvoid() {
         });
     }
 
+    ActionListener<Void> acquireSkipUnavailable(@Nullable String computeClusterAlias) {
+        if (computeClusterAlias == null
+            || esqlExecutionInfo.isCrossClusterSearch() == false
+            || runningOnRemoteCluster()
+            || computeClusterAlias.equals(RemoteClusterAware.LOCAL_CLUSTER_GROUP_KEY)
+            || esqlExecutionInfo.isSkipUnavailable(computeClusterAlias) == false) {
+            return acquireAvoid();
+        }
+        return refs.acquire().delegateResponse((l, e) -> {
+            LOGGER.error("Skipping unavailable cluster {} in ESQL query: {}", computeClusterAlias, e);
```

Comment:
My guess is that we probably don't want to log this at all. If we do, then it would likely be at debug level. Having this info in the logs. Search CCS does not log this for example.

Reply:
Yes, the logging is for now for debugging purposes, I'll drop it when the patch is finalized.